doc/doc-site/README.md

```diff
@@ -6,40 +6,32 @@ Econia's docs are built using [Docusaurus].
 
 1. Install [Homebrew] or a similar package manager.
 
-1. Install [Yarn]:
+2. Install [pnpm]:
 
    ```zsh
 
-   brew install yarn
+   wget -qO- https://get.pnpm.io/install.sh | sh -
```

Comment:
Can we switch this to brew to be consistent with other package install instructions?

Cam confirm that `brew install pnpm` is valid